src/core/agent/services/multiSigService.ts

```diff
@@ -879,26 +867,24 @@ class MultiSigService extends AgentService {
     };
   }
 
-  async endRoleAuthorization(multisigSignifyName: string): Promise<void> {
+  async endRoleAuthorization(multisigId: string): Promise<void> {
```

Comment:
This is called from KeriaNotificationService but there are no updates in that file. Until we have proper e2e tests in place, for these kinds of PRs you have to test things like creating a multi-sig and receiving a credential to that multi-sig

Reply:
So, Do we still use multisigSignifyName to call .addEndRole?